aikau/src/main/resources/alfresco/services/DocumentService.js

```diff
@@ -184,21 +184,10 @@ define(["dojo/_base/declare",
             var nodeRef = NodeUtils.processNodeRef(payload.nodeRef),
             targetNodeUri = nodeRef.uri;
 
-            // Construct the URI for the request...
```

Comment:
This code part was a remnant from a copy&paste from onRetrieveDocumentsRequest, but the calculated uriPart was not used at all.